### DISCUSSION TITLE
Add a custom finding template

### DISCUSSION THREAD

**Discussion by croclius** [Syslifters/sysreptor#354] [Q&A]
Hello there. I want my pentest findings to look like the screenshot below, but I am not able to do it in Sysreptor. Can you help me build it like that?
![image](https://github.com/user-attachments/assets/76986c59-c69b-46d6-8f08-ef8fd882d223)

**Comment by aronmolnar** [comment]
I suggest that you use an existing report design as the basis. "Demo Calzone" has a quite similar table structure, yet a little bit different items. You can update the table structure in the HTML section of the design.

![image](https://github.com/user-attachments/assets/31924172-678e-4967-9f68-614ed2783439)

To update the colors, you must adapt the CSS part of design.

![image](https://github.com/user-attachments/assets/14dd9951-e713-4681-94a4-1b2d98b00d0e)
